### DISCUSSION TITLE
友链的路由

### DISCUSSION THREAD

**Discussion by notmaster-C** [liuzhihang/halo-theme-hao#753] [Q&A]
# Q:友链中申请友链的路径是写死的，如果用其他插件的，比如连接管理插件，这个按钮在这很突兀还没用  。
   <div class="banner-button-group">
                <a class="banner-button secondary" data-pjax-state="" onclick="travelling()">
                    <i class="haofont hao-icon-paper-plane1"></i>
                    <span class="banner-button-text">随机访问</span>
                </a>
                <a class="banner-button" href="#post-comment" rel="external nofollow">
                    <i class="haofont hao-icon-link"></i>
                    <span class="banner-button-text">申请友链</span>
                </a>
            </div>
期望：可否适配，支持自定义🙏

**Comment by notmaster-C** [comment]
或者可以支持关闭显示这个地方也可以，目前主题是不支持关闭这个的🙏

**Comment by notmaster-C** [comment]
之所以问题题目叫做友链的路由，因为最近才使用halo，使用了这个主题以后，好多路由都找不到呢，比如我的装备的，按照别人的路由是进不去的，应该是没有设置插件吧

**Reply by carolcoral** [reply]
> 之所以问题题目叫做友链的路由，因为最近才使用halo，使用了这个主题以后，好多路由都找不到呢，比如我的装备的，按照别人的路由是进不去的，应该是没有设置插件吧

我的装备的路由是固定的，你看看reademe的插件中的描述
